autoware.repos

```diff
@@ -90,3 +90,8 @@ repositories:
     type: git
     url: https://github.com/autowarefoundation/autoware_individual_params.git
     version: main
+  # middleware
+  middleware/heaphook:
```

Comment:
Currently, non-AWF repositories are named `external`.

For the background, see https://github.com/autowarefoundation/autoware_core_universe_prototype/pull/374.

```suggestion
  middleware/external/heaphook:
```